test/support/utility.ex

```diff
@@ -258,17 +258,20 @@ defmodule Membrane.VideoCompositor.Test.Support.Utils do
   Compare contents of two raw files, succeeding when `different_bytes / length < allowed_error`
   """
   @spec compare_contents_with_error(binary(), binary(), float()) :: boolean()
-  def compare_contents_with_error(output_path, reference_path, allowed_error \\ 0.003) do
+  def compare_contents_with_error(output_path, reference_path, allowed_error \\ 1.5) do
```

Comment:
Nit: `allowed_mse` instead of `allowed_error` would be a little bit more descriptive.